<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExpenseWidget</class>
 <widget class="QWidget" name="ExpenseWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Расходы</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelDate">
       <property name="text">
        <string>Дата:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDateEdit" name="editDate">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="date">
        <date>
         <year>2000</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelCategory">
       <property name="text">
        <string>Категория:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboCategory"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelAmount">
       <property name="text">
        <string>Сумма:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="editAmount">
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelUser">
       <property name="text">
        <string>Пользователь:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboUser">
       <item>
        <property name="text">
         <string>Новый пользователь</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btnAddExpense">
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableExpense"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
